pyaedt/application/Variables.py

```diff
@@ -1236,6 +1238,81 @@ def delete_variable(self, var_name):
                 return True
         return False
 
+    @pyaedt_function_handler()
+    def used_variable(self, var_name):
```

Comment:
@SMoraisAnsys @Samuelopez-ansys I think we should rename this method `is_variable_used` because it just return a bool.